_posts/2015-04-25-basic-maintenance.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Ongoing maintanence and basic operations
+layout: post
+category: deployment
+permalink: /deployment/maintenance.html
+---
+
+## Configuration and logs
+
+The supervisor config can be found in `/opt/redash/supervisord/supervisord.conf`.
+
+There you can see the names of its programs (`redash_celery`, `redash_server`) and the location of their logs.
+
+## Restart
+
+### Restarting Celery workers
+
+1. stop celery:
+
+`sudo supervisorctl stop redash_celery`
+
+(celery might take some time to stop, if it's in the middle of running a query)
+
+2. flush redis 
```

Comment:
It's worth mentioning that flushing Redis will remove all pending queries, and only needed in case of an issue. In all other cases (like when upgrading) you should just run `sudo supervisorctl restart redash_celery`.